.env

```diff
@@ -89,7 +89,7 @@ TZ=UTC
 # Used through docker-compose.yml and serves as the default version for the
 # ci/scripts/install_vcpkg.sh script. Prefer to use short SHAs to keep the
 # docker tags more readable.
-VCPKG="943c5ef1c8f6b5e6ced092b242c8299caae2ff01"    # 2024.04.26 Release
+VCPKG="f7423ee180c4b7f40d43402c2feb3859161ef625"    # 2024.06.15 Release
```

Comment:
Could you also update the below `PYTHON_WHEEL_WINDOWS_IMAGE_REVISION`?
(We don't need to update `PYTHON_WHEEL_WINDOWS_TEST_IMAGE_REVISION`?)

Reply:
sure